### ISSUE TITLE
Please allow wildcards in file include statements

### ISSUE THREAD

**Issue by blais** [beancount/beancount#241]
**[Original report](https://bitbucket.org/blais/beancount/issue/241) by Martin Michlmayr (Bitbucket: [tbm13](https://bitbucket.org/tbm13), GitHub: [tbm](https://github.com/tbm)).**

----------------------------------------

Please support wildcards for file includes.  Currently I get:

```
2018.beancount:152:     Invalid token: 'loans/*/2018.beancount'
```

**Comment by blais** [comment]
**Original comment by Martin Michlmayr (Bitbucket: [tbm13](https://bitbucket.org/tbm13), GitHub: [tbm](https://github.com/tbm)).**

----------------------------------------

Sorry, the error was because I forgot to quote the string.  I get:

```
<load>:0:       File "/home/tbm/personal/finance/loans/*/2018.beancount" does not exist
```

**Comment by blais** [comment]
**Original comment by Jakob Schnitzer (Bitbucket: [yagebu](https://bitbucket.org/yagebu), GitHub: [yagebu](https://github.com/yagebu)).**

----------------------------------------

This is a duplicate of #110

**Comment by blais** [comment]
**Original comment by Martin Blais (Bitbucket: [blais](https://bitbucket.org/blais), GitHub: [blais](https://github.com/blais)).**

----------------------------------------

Issue #110 was marked as a duplicate of this issue.

**Comment by blais** [comment]
**Original comment by Martin Michlmayr (Bitbucket: [tbm13](https://bitbucket.org/tbm13), GitHub: [tbm](https://github.com/tbm)).**

----------------------------------------

I created pull request #59.

**Comment by blais** [comment]
**Original comment by Martin Blais (Bitbucket: [blais](https://bitbucket.org/blais), GitHub: [blais](https://github.com/blais)).**

----------------------------------------

Merged in tbm13/beancount/bug241 (pull request #59)

Fixed #241: allow wildcards in include statements

**Comment by blais** [comment]
**Original comment by Martin Blais (Bitbucket: [blais](https://bitbucket.org/blais), GitHub: [blais](https://github.com/blais)).**

----------------------------------------

Fixed #241: allow wildcards in include statements

**Comment by blais** [comment]
**Original comment by Martin Blais (Bitbucket: [blais](https://bitbucket.org/blais), GitHub: [blais](https://github.com/blais)).**

----------------------------------------

Merged in tbm13/beancount/bug241 (pull request #59)

Fixed #241: allow wildcards in include statements
